test/unit/algorithm_mode/test_custom_metrics.py

```diff
@@ -52,7 +53,19 @@ def test_multiclass_accuracy():
     assert accuracy_result == .5
 
 
+def test_multiclass_accuracy_softprob():
```

Comment:
I would add an explicit test for binary classification.

Reply:
Added